src/PKSim.Presentation/Presenters/Compounds/CreateEnzymaticProcessPresenter.cs

```diff
@@ -42,16 +44,33 @@ private IEnumerable<Compound> allPossibleMetabolites()
          return _buildingBlockRepository.All<Compound>()
             .OrderBy(x => x.Name);
       }
+      
+      protected override void Rebind(EnzymaticProcess enzymaticProcess, Compound compound)
```

Comment:
This is the actual code here. It is required to save the metabolite name before updating the view with thew new selected template

Reply:
This is really the only change in this code. All the rest is code reformatting using terser syntax and tests